kitchen.yml

```
@@ -297,14 +297,20 @@ suites:
         base:
           '*':
             - collectd._mapdata
+            - postgres.client
             - collectd
+            - collectd.postgresql
       pillars:
         top.sls:
           base:
             '*':
               - collectd
       pillars_from_files:
         collectd.sls: test/salt/default/pillar/collectd.sls
```

Let's take advantage of this PR to use a consistent location for this pillar file (compared to other formulas):

```suggestion
        collectd.sls: test/salt/pillar/default/collectd.sls
```

Obviously, the pillar file will need to be moved as well.

---

It seems to me that in some formulas it is the structure 'test/salt/default/pillar/' which is used.... As for example in nginx-formula : https://github.com/saltstack-formulas/nginx-formula/tree/master/test/salt/default